components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -137,6 +137,7 @@
 
     private boolean forceCaseInsensitiveComparisons = false;
     private boolean multiGroupAppSharingEnabled = false;
+    public static final String API_SCOPES_SHARING = "api-scopes-sharing";
```

Comment:
Shall we change this to ENABLE_API_SCOPE_SHARING? and let's move this to APIConstants class.

And let make the property "enable-api-scopes-sharing=true" so that one can disable this by "enable-api-scopes-sharing=false".